queries/make/highlights.scm

```diff
@@ -13,7 +13,22 @@
 (rule (targets
        (word) @function.builtin
        (#any-of? @function.builtin
+        ".DEFAULT"
+        ".SUFFIXES"
+        ".DEFAULT"
+        ".DELETE_ON_ERROR"
+        ".EXPORT_ALL_VARIABLES"
+        ".IGNORE"
+        ".INTERMEDIATE"
+        ".LOW_RESOLUTION_TIME"
+        ".NOTPARALLEL"
+        ".ONESHELL"
         ".PHONY"
+        ".POSIX"
+        ".PRECIOUS"
+        ".SECONDARY"
+        ".SECONDEXPANSION"
+        ".SILENT"
         ".SUFFIXES")))
```

Comment:
maybe they are `@constant.builtin` but I understand that `@function.builtin` has also its justification